### PR TITLE
Additional design updates for add buttons

### PR DESCRIPTION
**Pull request by dannychapman**
- Updated styling for "add parent" and "add kid" buttons - now full width
- Updated layout, optional and required labels for contact information (please review data entry still works correctly)
- Updated colors, styling for incomplete and complete sections

Please review @greggersh or @paulsmith

### CODE REVIEW

File: js/templates/adults.html [hunk 1]
```diff
@@ -30,18 +30,18 @@
   <div class="add-person">
     <div class="person add">
       <div class="person-name-bar button full-width">
-        <div class="person-name">Add another adult</div>
+        <!-- <div class="person-name"></div> -->
         <div class="actions">
-          <span aria-hidden="true">+</span> Add
+          <span aria-hidden="true">+</span> Add Add another adult
```
Comment (greggersh) [inline js/templates/adults.html]: I haven't looked at this but did you mean to double the "Add"?

Comment (dannychapman) [inline js/templates/adults.html]: Nope, good catch. Fixing.

File: sass/screen.css [hunk 2]
```diff
@@ -452,9 +452,6 @@ a {
 /* =============
    HEADERS
    ============= */
-#menu:before {
-  content: "🍔 "; }
```
Comment (greggersh) [inline sass/screen.css]: So long, hamburger.

**Comment by greggersh** [comment]
Looks good to me.

**Comment by paulsmith** [comment]
LGTM

**Pull request closed without merging**